routes/index.js

```diff
@@ -0,0 +1,74 @@
+var express = require('express');
+var router = express.Router();
+
+var Pirate = require('../models/pirate');
+
+// GET
+router.get('/', function(req, res, next){
+  // index view
+  Pirate.find({}, function(err, pirates){
+    if (err) console.log(err);
+    res.render('index', { title: 'Pirate Information', pirates: pirates });
+  })
+});
+
+// NEW
+router.get('/new', function(req, res, next){
+  // get new recipe form
+  res.render('new');
+});
+
+
+// CREATE
+router.post('/', function(req, res, next){
+  // redirect to index
+  var newPirate = new Pirate({
+    name: req.body.name,
+    catchphrase: req.body.catchphrase
+  });
+
+  newPirate.save(function(err, pirate){
+    if (err) console.log(err);
+    res.redirect('/')
+  })
+});
+
+// SHOW
+router.get('/:id', function(req, res, next){
+  var id = req.params.id;
+  Pirate.findOne({_id: id }, function(err, pirate){
+    if (err) console.log(err);
+    res.render('show', {pirate: pirate})
+  })
+});
+
+// EDIT
+router.get('/:id/edit', function(req, res, next){
+  // get edit recipe form
+  var id = req.params.id;
+  Pirate.findOne({_id: id }, function(err, pirate){
+    if (err) console.log(err);
+    res.render('edit', {pirate: pirate})
+  })
+});
+
+// UPDATE
+router.patch('/:id', function(req, res, next){
+  // redirect to show
```

Comment:
In the above EDIT and SHOW routes, you are getting the id from `req.params.id` and setting it to a variable. I would be consistent and either always set it to that variable, or just put it directly into the query like you are in the UPDATE route.